lib/packages/ui/lib/src/widgets/media/image.dart

```diff
@@ -79,19 +82,15 @@ class _LMImageState extends State<LMChatImage> {
   }
 
   @override
-  void didUpdateWidget(covariant LMChatImage oldWidget) {
-    super.didUpdateWidget(oldWidget);
-    style = widget.style ?? LMChatTheme.theme.imageStyle;
+  void dispose() {
+    super.dispose();
   }
 
   @override
   Widget build(BuildContext context) {
-    return AbsorbPointer(
-      absorbing: widget.onTap == null,
-      child: GestureDetector(
-        onTap: () => widget.onTap?.call(),
-        child: _buildImageWidget(),
-      ),
+    return GestureDetector(
+      onTap: () => widget.onTap?.call(),
```

Comment:
can we change it like 
`onTap: widget.onTap,`

Reply:
Done